src/services/BlockSynchronizer.ts

```diff
@@ -74,15 +74,42 @@ class BlockSynchronizer {
 
     const [leavesSynchronizers, blockIds] = await this.processBlocks(chainStatusNextBlockId, validators, mongoBlocks);
 
-    if (blockIds.length > 0) {
-      this.logger.info(`Synchronized leaves for blocks: ${blockIds.join(',')}`);
-      const updated = await this.updateSynchronizedBlocks(await Promise.all(leavesSynchronizers), blockIds);
-      const success = updated.updatedFinalizedBlocks;
-      const failed = updated.updatedFailedBlocks;
-      this.logger.info(`Finalized successfully/failed: ${success}/${failed}. Total: ${success + failed}`);
-    } else {
-      this.logger.info('nothings was synchronised');
+    if (!blockIds.length) {
+      return;
```

Comment:
should probably leave the log message here

Reply:
done. more detailed message will be produced by `processBlocks()`